DataConnectors/AWS-S3/Utils/AwsResourceCreator.ps1

```diff
@@ -23,7 +23,7 @@ function New-ArnRole
             Write-Output "`n`n"
             Write-Log "You must specify the the Azure Sentinel Workspace ID. This is found in the Azure Sentinel portal." -LogFileName $LogFileName -Severity Information -LinePadding 1
             
-            $workspaceId = Read-ValidatedHost -Prompt "Please enter your Azure Sentinel Workspace ID (External Id)"
+            $workspaceId = Read-ValidatedHost -Prompt "Please enter your Azure Sentinel External ID (Workspace ID)"
```

Comment:
Should this be: "Please enter the External Id (Azure Sentinel Workspace ID)"

I think it is clear as it is, but just want to make sure.

Reply:
I think it is clear. 
this text is the same as we have in the UI and there is the explanation comment above, and